app/src/main/java/com/topjohnwu/magisk/core/Config.kt

```diff
@@ -179,7 +184,6 @@ object Config : PreferenceConfig, DBConfig {
         prefs.edit {
             // Settings migration
             if (prefs.getBoolean(SU_FINGERPRINT, false))
-                suBiometric = true
             remove(SU_FINGERPRINT)
```

Comment:
Maybe we can just remove all stuffs about the `SU_FINGERPRINT` field, since its functionality has been removed in b29f0ca which is 5 years ago?